app/app.go

```diff
@@ -716,7 +720,7 @@ func New(
 			NoBaseFee:                false,
 			BaseFeeChangeDenominator: 100000000,
 			ElasticityMultiplier:     2,
-			InitialBaseFee:           initialBaseFee,
+			BaseFee:                  sdk.NewInt(initialBaseFee),
```

Comment:
I think we should remove this upgrade handler, since it shouldn't be used as the target of `v0.7.0` upgrade anymore.